modules/server/src/network/aggregations/index.ts

```diff
@@ -1,22 +1,39 @@
 import { SupportedAggregation, SUPPORTED_AGGREGATIONS } from '../common';
 import { Aggregations, NetworkAggregation, NumericAggregations } from '../types';
 
-// TODO: implement
-export const resolveAggregations = (networkResults: void) => null;
+/**
+ * Pick each field from network result aggregations and reduce into single aggregation
+ *
+ * @param networkResults
+ * @param rootQueryFields
+ */
+export const resolveAggregations = (networkResults, rootQueryFields) => {
+	const resolvedNetworkAggregations = rootQueryFields.map((fieldName) => {
+		const fieldAggregations = networkResults.map((networkResult) => {
+			const documentName = Object.keys(networkResult)[0];
+			return networkResult[documentName][fieldName];
+		});
+		const aggregationType = fieldAggregations[0].__typename;
```

Comment:
Any potential for empty arrays / need for falsy handling ?

Reply:
at this point falseys should be filtered out and every apollo gql server returns `__typename`